apps/web/pages/index.tsx

```diff
@@ -1,11 +1,30 @@
-import { Button, print } from "ui";
+import Link from "next/link";
+import { Button } from "ui";
+import { Header } from 'shared/components'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/gabrieldocoutos/papaya/pullRequest?prid=8595723)
```suggestion
import { Header } from "shared/components"
```